src/docs/product/integrations/jira/index.mdx

```diff
@@ -86,7 +84,9 @@ openssl x509 -pubkey -noout -in jira_publickey.cer  > jira_publickey.pem
 
 Jira should now be authorized for all projects under your Sentry organization.
 
-## Issue Management
+## Configure
+
```

Comment:
missing something here? It's generally unusual to move from a heading directly to another heading without some sort of intro material.